options.go

```diff
@@ -39,6 +39,7 @@ type Options struct {
 	ClientSecret    string `flag:"client-secret" cfg:"client_secret" env:"OAUTH2_PROXY_CLIENT_SECRET"`
 	TLSCertFile     string `flag:"tls-cert-file" cfg:"tls_cert_file" env:"OAUTH2_PROXY_TLS_CERT_FILE"`
 	TLSKeyFile      string `flag:"tls-key-file" cfg:"tls_key_file" env:"OAUTH2_PROXY_TLS_KEY_FILE"`
+	ProxyTimeOut    int    `flag:"proxy-time-out" cfg:"proxy_time_out" env:"OAUTH2_PROXY_TIME_OUT"`
```

Comment:
To be consistent with other options, the env should be
```suggestion
	ProxyTimeOut    int    `flag:"proxy-time-out" cfg:"proxy_time_out" env:"OAUTH2_PROXY_PROXY_TIME_OUT"`
```

Also, I wonder if this could be a `time.Duration` rather than an `int`, we have other timeouts configured as `time.Duration` (eg `FlushInterval`), would remove the cast where this is used and allow users to specify a duration in the args like `30s` or `2m` for 30 seconds and 2 minutes respectively

Reply:
I will do this change